### PR TITLE
PACT-725 Fix pagination direction link

### PR DESCRIPTION
**Pull request by felixgb**

### CODE REVIEW

**Review by rwalpole — APPROVED** [review]

File: test/controllers/EditSetControllerSpec.scala [hunk 1]
```diff
@@ -651,6 +651,88 @@ class EditSetControllerSpec extends BaseSpec {
...
+
+        assertPageAsExpected(
+          asDocument(page),
+          ExpectedEditSetPage(
+            title = "Browse Edit Set (Page 1 of 2)",
+            caption = "Showing 1 - 10 of 12 records",
+            header = "Edit set: COAL 80 Sample",
+            ccrTableHeader = ExpectedTableHeader("ccr", "ascending"),
+            scopeAndContentTableHeader = ExpectedTableHeader("scope-and-content", "ascending"),
+            coveringDateTableHeader = ExpectedTableHeader("covering-dates", "ascending"),
```
Comment (tna-erasmos) [inline test/controllers/EditSetControllerSpec.scala:667]: Likely out of scope, but I just noticed that we cannot tell which of the headers is the "active" one (that is, ordered). However, from the order of the records, it's clear.

File: test/services/EditSetRowOrderSpec.scala [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.gov.nationalarchives.omega.editorial.services
```
Comment (tna-erasmos) [inline test/services/EditSetRowOrderSpec.scala:22]: The package should only be 'services', no?

File: test/services/EditSetRowOrderSpec.scala [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+import support.BaseSpec
+import uk.gov.nationalarchives.omega.editorial.services.EditSetEntryRowOrder
```
Comment (tna-erasmos) [inline test/services/EditSetRowOrderSpec.scala:25]: Unused.

File: test/services/EditSetRowOrderSpec.scala [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+class EditSetEntryRowOrderSpec extends BaseSpec {
```
Comment (tna-erasmos) [inline test/services/EditSetRowOrderSpec.scala:27]: Did you want to match the filename?

File: app/uk/gov/nationalarchives/omega/editorial/services/EditSetPagination.scala [hunk 5]
```diff
@@ -57,10 +57,10 @@ class EditSetPagination(
         (None, None)
 
       case 1 =>
-        (None, nextLink(numberOfPages))
+        (None, nextLink(currentPage + 1))
```
Comment (tna-erasmos) [inline app/uk/gov/nationalarchives/omega/editorial/services/EditSetPagination.scala:60]: Was this fix related to a different bug? I didn't see any relevant changes to the tests.

File: app/uk/gov/nationalarchives/omega/editorial/services/EditSetPagination.scala [hunk 6]
```diff
@@ -57,10 +57,10 @@ class EditSetPagination(
...
 
       case `numberOfPages` =>
-        (previousLink(1), None)
+        (previousLink(currentPage - 1), None)
```
Comment (tna-erasmos) [inline app/uk/gov/nationalarchives/omega/editorial/services/EditSetPagination.scala:63]: Was this fix related to a different bug? I didn't see any relevant changes to the tests.

**Review by tna-erasmos — APPROVED** [review]
Minor points.

**Comment by tna-erasmos** [comment]
I QA'd this, and all works as it should.

File: test/controllers/EditSetControllerSpec.scala (hunk 1 above)
Comment (felixgb) [reply test/controllers/EditSetControllerSpec.scala:667]: Hm, we could add the CSS selectors for the `aria-sort` attribute here.

**Pull request merged**